src/internal/interpretor.js

```diff
@@ -29,7 +29,7 @@ export default function interpretor(rej, res){
   function settle(m){
     settled = true;
     future = m;
-    if(future._spawn){
+    if(future && future._spawn){
```

Comment:
Why this line should be reverted: There is no need to test for the presence of `future` here. We have *just* assigned the function argument `m` to `future`. Since it is our intention to call this function with an instance of Future always, `future` should always be an there. Besides, if it's not here then our interpretor will fail in the next tick because it's `settle`s job to prepare an instance of `future` to be used in the next tick.